### DISCUSSION TITLE
Feature Request:  Select item of search results and complete textbox with this.

### DISCUSSION THREAD

**Discussion by tamutamu** [Wox-launcher/Wox#3943] [General]
**Is your feature request related to a problem? Please describe.**
 - There are times when you want to copy search result item or edit this.
 - In any plugin, I want to sync textbox by selecting item of search results.

**Describe the solution you'd like**
 1. Searching by keyword.
 1. Select item with result pane.
 1. For example, By press tab key, complete selected item path, name, etc.. in textbox.

**Comment by TuRDMaN** [comment]
This feature may be a dealbreaker for me. With other similar programs, while I am typing a path, if I see something I want in the results, I can tap the down arrow to highlight that result, then right arrow/tab to fill in the search box with that result, then can continue typing, copy the path, etc.

I really miss this functionality, and Wox just feels broken without it.

**Comment by qianlifeng** [comment]
Already implemented in v2
